### PR TITLE
feat(query): [BACK-1558] Create a query to get a curated item by its `externalId`

### PR DESCRIPTION
**Pull request by nina-py**
## Goal

Add a new query to unblock creating individual pages for curated items on the frontend.

- Added a new query to schema with a filter on `externalId`.

- Added integration tests.

## References

JIRA ticket:
* https://getpocket.atlassian.net/browse/BACK-1558

### CODE REVIEW

File: schema-admin.graphql [hunk 1]
```diff
@@ -862,6 +862,16 @@ type Query {
...
+    """
+    Retrieves an approved item with the given external ID.
+    """
+    approvedCorpusItem(
+        """
+        ID of the approved item. A string in UUID format.
+        """
+        externalId: ID!
+    ): ApprovedCorpusItem
+
```
Comment (nina-py) [inline schema-admin.graphql:874]: Note the new naming convention: no `get` in query name.

File: src/admin/resolvers/queries/sample-queries.gql.ts [hunk 2]
```diff
@@ -81,6 +81,15 @@ export const GET_APPROVED_ITEM_BY_URL = gql`
...
 
+export const GET_APPROVED_ITEM_BY_EXTERNAL_ID = gql`
+  query approvedCorpusItem($externalId: ID!) {
+    approvedCorpusItem(externalId: $externalId) {
+      ...CuratedItemData
+    }
+  }
+  ${CuratedItemData}
+`;
+
```
Comment (nina-py) [inline src/admin/resolvers/queries/sample-queries.gql.ts:92]: It felt right to keep the `GET_` prefix in the sample query name - need an action verb here.

**Review by Herraj — APPROVED** [review]

File: schema-admin.graphql [hunk 3]
```diff
@@ -862,6 +862,16 @@ type Query {
         url: String!
     ): ApprovedCorpusItem
 
+    """
+    Retrieves an approved item with the given external ID.
+    """
+    approvedCorpusItem(
```
Comment (bassrock) [inline schema-admin.graphql:868]:
```suggestion
    approvedCorpusItemByExternalId(
```

Comment (nina-py) [reply schema-admin.graphql]: Done! We can use this and keep `approvedCorpusItem` available for when `@oneof` directive is available as discussed at today's pairing: https://github.com/graphql/graphql-spec/pull/825#issuecomment-938051965

**Review by Herraj — APPROVED** [review]
🚀

**Pull request merged**